### ISSUE TITLE
[Feature Request] Any way to delete the search & watched history?

### ISSUE THREAD

**Issue by shikiroot** [tatsuz0u/EhPanda#154]
Thanks for this great project, very useful. So I don't know how to delete the search history and watched history in sidebar, or it doesn't support it yet?

**Comment by tatsuz0u** [comment]
Yes, currently there's no way to delete search & gallery view histories. I'll add this feature to my TODO list. Thanks for bringing this up.

**Comment by tatsuz0u** [comment]
Done. 7dfcca5 aa8a6c2

**Comment by tatsuz0u** [comment]
Done. 7dfcca5 aa8a6c2
